scratch3/blocks.js

```diff
@@ -510,6 +510,7 @@ BlockView.prototype.draw = function() {
   // Commands have a minimum width
   // The hat min-width is arbitrary (not sure of Scratch 3 value)
   // Outline min-width is deliberately higher (because Scratch 3 looks silly)
+  var orgInnerWidth = innerWidth
```

Comment:
What's `org` short for? Original? Maybe `originalInnerWidth` would be better :-)